bip-0373.mediawiki

```diff
@@ -45,17 +45,18 @@ The new per-input types are defined as follows:
 |-
 | rowspan="2"|MuSig2 Participant Public Keys
 | rowspan="2"|<tt>PSBT_IN_MUSIG2_PARTICIPANT_PUBKEYS = 0x1a</tt>
-| <tt><33 byte plain aggregate pubkey></tt>
-| <tt><33 byte compressed pubkey>*</tt>
+| <tt><33 byte aggregate pubkey (compressed)></tt>
+| <tt><33 byte participant pubkey (compressed)>*</tt>
 | rowspan="2"|
 | rowspan="2"|
 | rowspan="2"| 0, 2
 |-
-| The MuSig2 aggregate plain public key<ref>'''Why the plain aggregate public key instead of x-only?'''
+| The MuSig2 aggregate public key (compressed) <ref>'''Why the compressed aggregate public key instead of x-only?'''
 BIP 32 requires public keys to include their evenness byte. Aggregate public keys are expected to be
 derived from, following [[bip-0328.mediawiki|BIP 328]], and therefore will
 need to include the evenness. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
-to identify master keys, and these fingerprints require full compressed public keys. By including
+to identify master keys, and these fingerprints require y-coordinate of the public key,
```

Comment:
nit

```suggestion
to identify master keys, and these fingerprints require the y-coordinate of the public key,
```

Reply:
The previous sentence is confusing as well:

"Aggregate public keys are expected to be derived from, following BIP 328, and therefore [...]"

To a layman (me!) it reads like there's a word missing ("derived from... what?"). But I see now that it's referring to them as the starting point ("from" the aggregate public key to a BIP32 public key). Perhaps clearer to say:

"BIP 32 public keys are expected to be derived from aggregate public keys (per BIP 328) and therefore..."

Reply:
> The previous sentence is confusing as well:

@kdmukai WDYT about

```diff
-BIP 32 requires public keys to include their evenness byte. Aggregate public keys are expected to be
-derived from, following [[bip-0328.mediawiki|BIP 328]], and therefore will
-need to include the evenness. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
+BIP 32 requires public keys to include their evenness byte. Per [[bip-0328.mediawiki|BIP 328]], BIP 32
+public keys can be derived from a BIP 327 MuSig2 aggregate public key; therefore, the latter needs
+to include the evenness byte as well. Furthermore, PSBT_IN_TAP_BIP32_DERIVATION fields include fingerprints
```

Reply:
@jonatack YES! Much clearer. Minor edit suggestion: swap and slightly rework the first two sentences:

"BIP 32 public keys can be derived from a BIP 327 MuSig2 aggregate public key (see: BIP 328). But since BIP 32 requires public keys to include their evenness byte, BIP 327 MuSig2 aggregate public keys should ("must"?) include their evenness byte as well."

_(minor minor note: I tend to avoid "latter" / "former" constructions as for some reason my brain just finds them extra taxing to understand. The repetition I suggest above is a little annoying but prioritizes absolute clarity)_

Reply:
@kdmukai LGTM (with "must", I believe). Can you open a pull?

Reply:
Resolving this thread; done in #1717.